docs/preview/02-Features/powershell/arm.md

```diff
@@ -64,10 +64,12 @@ It is possible to supply injection instructions in the injection annotation, thi
 | `InjectAsJsonObject`  | Tests if the content is valid JSON and makes sure the content is injected without surrounding double quotes |
 
 Usage of multiple injection instructions is supported as well, for example if you need both the `EscapeJson` and `ReplaceSpecialChars` functionality.
+The reference to the file to inject can either be a path relative to the 'parent' file or an absolute path.
```

Comment:
I'm wondering if '... path relative to the parent file...' is clear enough. Maybe @pim-simons can shed some light here.

Reply:
Combined with the examples on line 71 to 74 this is clear enough for me 👍🏻

Reply:
Ok, great!